articles/language/index.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: The Q# Programming Language | Microsoft Docs 
-description: The Q# Programming Language
+title: The Q# Programming Language
+description: Introduction to the Q# programming language for quantum programs. 
```

Comment:
The phrase "programming language for quantum programs" feels fairly redundant to me?